### PR TITLE
docs: Clean-up opinion and make the position clearer

### LINKED ISSUE #90: Should the README's position statement on unsafe code be adjusted for the new ownership?
The README says the following:

> It is however the opinion of the author of this project that libraries choosing to abstain from unsafe Rust usage when possible should be promoted.

Now that the crate is owned by the Secure Code Working Group, I suggest either identifying "the author" or changing the sentence to state the opinion of the Working Group, whichever you consider appropriate.

### PR DESCRIPTION
**Pull request by pinkforest**
Closes #90

### CODE REVIEW

**Comment by pinkforest** [comment]
@8537 feedback would be welcome

I'm also trying to address this a bit:
https://github.com/rust-secure-code/cargo-geiger/discussions/302#discussioncomment-2753511

**Comment by 8573** [comment]
I think the opinions of @rust-secure-code will be more valuable and relevant than mine, but I'll leave some questions.

File: README.md [hunk 1]
```diff
@@ -7,12 +7,27 @@ cargo-geiger ☢️
...
+This project is intended as an attempt to:
+* Create pressure against __unnecessary__ usage of unsafe Rust
+
+This project is not intended as an attempt to:
+* Create pressure against __necessary__ usage of unsafe Rust
+
+There are several proposals / projects to qualify the statistics in more end-user usable form:
+* Safety Cross-Checking Human Validations - https://github.com/rust-secure-code/cargo-geiger/discussions/303
+
 This cargo plugin were originally based on the code from two other projects:
```
Comment (8573) [inline README.md:28]:
```suggestion
This Cargo plugin was originally based on the code from two other projects:
```

May as well fix the previous README's grammar here

File: README.md [hunk 2]
```diff
@@ -40,18 +55,6 @@ Output example
 
 ![Example output](https://user-images.githubusercontent.com/3704611/53132247-845f7080-356f-11e9-9c76-a9498d4a744b.png)
 
-Why even care about unsafe Rust usage?
---------------------------------------
-
-When and why to use unsafe Rust is out of scope for this project; it is simply
```
Comment (8573) [inline README.md:46]: Was it intended to drop the statement "When and why to use unsafe Rust is out of scope for this project"?

File: README.md [hunk 3]
```diff
@@ -40,18 +55,6 @@ Output example
...
-a tool that provides information to aid auditing and hopefully to guide
-dependency selection. It is, however, the opinion of the author of this project
-that __libraries choosing to abstain from unsafe Rust usage when possible should
-be promoted__.
```
Comment (8573) [inline README.md:50]: > It is, however, the opinion of the author of this project that __libraries choosing to abstain from unsafe Rust usage when possible should be promoted__.

I understand dropping altogether this statement of opinion, as a resolution of #90, to be the core point of this patch.  Insofar as my opinion is relevant, I have no objection to the patch overall, but I await the opinions of tarcieri and Shnatsel on whether this statement should be dropped entirely.

Comment (pinkforest) [reply README.md:50]: It's not dropped, it's moved.

File: README.md (hunk 2 above)
Comment (pinkforest) [reply README.md:46]: It was moved - it's redundant to point out what is not in scope or not - it's now rather generic statement without it's own heading at top.

File: README.md (hunk 1 above)
Comment (pinkforest) [reply README.md:28]: Thanks!

File: README.md [hunk 4]
```diff
@@ -7,12 +7,27 @@ cargo-geiger ☢️
...
 
-A program that lists statistics related to the usage of unsafe Rust code in a Rust
+A tool that lists statistics related to the usage of unsafe Rust code in a Rust
 crate and all its dependencies.
 
+These statistics currently provide blunt information to aid auditing, e.g:
+* https://github.com/rust-secure-code/safety-dance
+* https://github.com/crev-dev/cargo-crev
+
+This tool is currently not meant to advise directly whether the code ultimately is truly unsound or not.
```
[the remark this answers is not in the document]
Comment (pinkforest) [reply README.md:17]: I just replaced it with insecure. Thanks

File: README.md [hunk 5]
```diff
@@ -7,12 +7,27 @@ cargo-geiger ☢️
 [![crates.io](https://img.shields.io/crates/v/cargo-geiger.svg)](https://crates.io/crates/cargo-geiger)
 [![Crates.io](https://img.shields.io/crates/d/cargo-geiger?label=cargo%20installs)](https://crates.io/crates/cargo-geiger)
 
-A program that lists statistics related to the usage of unsafe Rust code in a Rust
+A tool that lists statistics related to the usage of unsafe Rust code in a Rust
 crate and all its dependencies.
 
+These statistics currently provide blunt information to aid auditing, e.g:
```
[the remark this answers is not in the document]
Comment (pinkforest) [reply README.md:13]: It is blunt information, it is not auditing information.

File: README.md [hunk 6]
```diff
@@ -7,12 +7,27 @@ cargo-geiger ☢️
...
+This tool is currently not meant to advise directly whether the code ultimately is truly unsound or not.
+
+This project is intended as an attempt to:
+* Create pressure against __unnecessary__ usage of unsafe Rust
+
+This project is not intended as an attempt to:
+* Create pressure against __necessary__ usage of unsafe Rust
+
+There are several proposals / projects to qualify the statistics in more end-user usable form:
+* Safety Cross-Checking Human Validations - https://github.com/rust-secure-code/cargo-geiger/discussions/303
```
[the remark this answers is not in the document]
Comment (pinkforest) [reply README.md:26]: I replaced it with Unsfe block Corss-Checking and Tracking, hopefully works better

**Pull request merged**